packages/react-jss/tests/theming.js

```diff
@@ -43,15 +48,11 @@ describe('React-JSS: theming', () => {
   const ThemedDynamicComponent = injectSheet(themedDynamicStyles)()
 
   let localJss
+  let generateId
 
   beforeEach(() => {
```

Comment:
somehow we ended up having 2 beforeEach in one describe, I think it wasn't intentional

Reply:
I checked, there are not two generateId's